tests/conformance_ruby/tags/include_tag_test.rs

```diff
@@ -81,33 +82,41 @@ fn test_include_tag_for() {
 }
 
 #[test]
-#[should_panic] // liquid-rust#237
+fn test_include_tag_with_local_variable() {
+    assert_template_result!(
+        "Single Locale: test123",
+        "{% include 'single_locale_variable' echo1:'test123' %}",
+        o!({}),
+        liquid()
+    );
+}
+
+#[test]
+#[should_panic]
 fn test_include_tag_with_local_variables() {
     assert_template_result!(
-        "Locale: test123 ",
-        "{% include 'locale_variables' echo1: 'test123' %}",
+        "Locale: test123",
+        "{% include 'locale_variables' echo1:'test123' %}",
```

Comment:
There are still whitespace changes.  I'd recommend reverting the file and just remove the `should_panic`s.